From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 25 Mar 2020 09:44:39 -0700
Subject: [PATCH] Do not disable unavailable plugin

Latest DNF complains about the `spacewalk` plugin being unavailable when
called by mock. Since it never was available in Clear, silence the
warning.

Rebased for mock 2.12 update by Patrick McCarty.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mock/py/mockbuild/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mock/py/mockbuild/config.py b/mock/py/mockbuild/config.py
index be686c1..95575ce 100644
--- a/mock/py/mockbuild/config.py
+++ b/mock/py/mockbuild/config.py
@@ -311,7 +311,7 @@ def setup_default_config_opts(unprivUid, unprivGid=None):
     config_opts['microdnf_common_opts'] = []
     config_opts['rpm_command'] = '/bin/rpm'
     config_opts['rpmbuild_command'] = '/usr/bin/rpmbuild'
-    config_opts['dnf_disable_plugins'] = ['local', 'spacewalk', 'versionlock']
+    config_opts['dnf_disable_plugins'] = ['local', 'versionlock']
     config_opts['user_agent'] = "Mock ({{ root }}; {{ target_arch }})"
     config_opts['opstimeout'] = 0
 
